docs/yml.md

```diff
@@ -342,6 +342,15 @@ dns_search:
   - dc2.example.com
 ```
 
+### devices
+
+Devices option.  List of device mappings.  Uses the same format as the --device docker client create option.
```

Comment:
- First sentence is unnecessary
- `--device` should be in backticks
- Wrap to 80 chars please!